pkg/golinters/gosec.go

```diff
@@ -22,24 +22,62 @@ import (
 
 const gosecName = "gosec"
 
-func NewGosec(settings *config.GoSecSettings) *goanalysis.Linter {
-	var mu sync.Mutex
-	var resIssues []goanalysis.Issue
+// see gosec#Config.convertGlobals
+func convertGosecGlobals(settings *config.GoSecSettings, conf gosec.Config) {
```

Comment:
Could you move helper functions `convertGosecGlobals` and `toGosecConfig` below `NewGosec`? Usually, unexported functions are below exported.

Reply:
updated, PTAL